### DISCUSSION TITLE
Facing rabbitmq connectivity issue from Jboss server.

### DISCUSSION THREAD

**Discussion by Vasanth03-dot** [rabbitmq/rabbitmq-server#3830] [Other]
Thank you for using RabbitMQ.

**STOP NOW AND READ THIS** BEFORE OPENING A NEW ISSUE ON GITHUB

Unless you are CERTAIN you have found a reproducible problem in RabbitMQ or
have a **specific, actionable** suggestion for our team, you must first ask
your question or discuss your suspected issue on the mailing list:

https://groups.google.com/forum/#!forum/rabbitmq-users

Team RabbitMQ does not use GitHub issues for discussions, investigations, root
cause analysis and so on.

Please take the time to read the CONTRIBUTING.md document for instructions on
how to effectively ask a question or report a suspected issue:

https://github.com/rabbitmq/rabbitmq-server/blob/master/CONTRIBUTING.md#github-issues

Following these rules **will save time** for both you and RabbitMQ's maintainers.

Thank you.

**Comment by Vasanth03-dot** [comment]
Below is the Error from server log:
2021-12-02 12:54:22,007 WARN  [org.jboss.jca.core.connectionmanager.pool.strategy.OnePool] (default task-1) IJ000604: Throwable while attempting to get a new connection: null: javax.resource.ResourceException: Error during setup 	at org.apache.qpid.ra.QpidRAManagedConnection.<init>(QpidRAManagedConnection.java:151) 	at org.apache.qpid.ra.QpidRAManagedConnectionFactory.createManagedConnection(QpidRAManagedConnectionFactory.java:160) 	at org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreConcurrentLinkedDequeManagedConnectionPool.createConnectionEventListener(SemaphoreConcurrentLinkedDequeManagedConnectionPool.java:1322)

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
The trace only says there was a null (JNDI resource?) pointer from a Qpid client. We don't have much else to work with. This seems to be a client configuration issue.

We don't use Qpid client in any of the tests. If it implements AMQP 1.0, that plugin must be enabled. For AMQP 0-9-1 or RabbitMQ streams, there are clients provided by our team.

RabbitMQ [logs all connection events](https://www.rabbitmq.com/logging.html#logged-events) as long as there is at least one byte sent on the connection. There are also guides that
explain how to efficiently narrow down [connectivity issues](https://www.rabbitmq.com/troubleshooting-networking.html) and [TLS connectivity issues](https://www.rabbitmq.com/troubleshooting-ssl.html).
